### ISSUE TITLE
Make clear how to solve unsynced OTPs

### ISSUE THREAD

**Issue by lukket** [raivo-otp/ios-application#204]
**Describe the bug**
When I go to the settings the synchronization status says ”1 unsynced OTP“. But it’s totally unclear which OTP it is and what I can do to solve this problem. It’s not even clear if there is really one unsynced OTP.

**Expected behavior**
A clear and concise description of what I as a user can do to make all OTPs synced correctly.

**Screenshots**
<img width="414" alt="Screenshot 2023-01-08 at 22 48 15 png" src="https://user-images.githubusercontent.com/2588942/211221003-9578567b-8afc-4d90-9ac9-ca7e57064854.png">

**Smartphone (please complete the following information):**
 - Device: iPhone 11
 - OS: iOS 16.2
 - Version 1.4.8

**Comment by tijme** [comment]
Hmm. Does one of your OTP's has a lightning icon next to it in your OTP list? ⚡

**Comment by lukket** [comment]
No, I can’t see any entry with a lightning icon. Although I have some entries with very long names. Is the icon also visible for entries with names that are longer than the row can display?

**Comment by tijme** [comment]
It should be, yes! How many OTP's do you have? I know that it will resync when you try to save the unsynced OTP. But if you have many, that can take a while to try all..

**Comment by lukket** [comment]
I went through all entries step by step. First the ones with very long names but nothing changed. Then I found two entries that had the same name and changed one of them. Now the synchronization is up-to-date again. I changed the name back again so both entries have the same name again, but still everything seems to be fine now 👍.

Could the duplicated name be the issue?

**Comment by LawHoo** [comment]
@tijme @lukket I have the same issue, but I don't have any entries with duplicate names. However, I have nearly 200 entries, and editing and re-saving all of them would be a huge pain. .... It would be nice to have a way to force a re-sync--that is, forcibly overwrite whatever is in iCloud with whatever is saved on the current device (with appropriate warnings about overwriting any unsynced changes from other devices). And, of course, it would be helpful if we could actually see what entry is triggering the unsynced issue. ... Thanks for a great app! :-)
